### DISCUSSION TITLE
Option to hide sender address and alias name in forwarded emails

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#435] [Feature Requests]
SimpleLogin has the ability to encrypt email content and subject with GPG, and to have a dummy subject. However, the sender of the email and the name of the alias that received it are not protected at all.

Let's say you are using an untrusted email provider to receive emails via SimpleLogin. As it is currently, you can make it so that the content and subject are hidden from the provider, but this provider can still see the original sender (the "From" field on most email clients) and the alias name (which would be the "To" field). This could be a problem if you were for example using a "professional" email to receive personal emails through aliases, as the person/company/etc you are sending/receiving email from is obvious from the sender, and likely also from the alias (unless you use gibberish as the alias name).

One way to fix this would be to have an option to only show "SimpleLogin" as the sender in the "From" field, and an option to make the mailbox the alias points to as the "To" field. The subject and alias are already shown in the message body with the mentioned existing options on, so the true sender could also be added to this (see below)

Current
```
From:    "Fancy Name - senderemail at sender.tld"
To:      aliasname.randomword@domain.tld
Subject: Encrypted Email

(PGP encrypted body)
Forwarded by SimpleLogin to aliasname.randomword@domain.tld with "email subject" as subject
```

Proposed example
```
From:    "SimpleLogin"
To:      realmailbox@untrusted.tld
Subject: Encrypted Email

(PGP encrypted body)
Forwarded from "Fancy Name - senderemail at sender.tld" by SimpleLogin to aliasname.randomword@domain.tld with "email subject" as subject
```

**Comment by nguyenkims** [comment]
That's indeed a good idea to add an additional option, similar to the "Hide email subject when PGP is enabled" one but for `From:` and `To:` field.

![image](https://user-images.githubusercontent.com/3167780/115237636-1302b480-a11d-11eb-96f5-c8704e0cddbc.png)

**Comment by shakedmartini** [comment]
This is quite similar: https://www.reddit.com/r/Simplelogin/comments/n937lw/can_i_hide_the_sender_from_my_mail_provider/

**Comment by mvevitsis** [comment]
Is it still not possible to hide the alias name? You can hide the sender name I see.
